app/assets/stylesheets/layout_components/forms.scss

```diff
@@ -366,98 +361,95 @@ input[type="checkbox"]{
   }
 }
 
-.date-pair{
+.date-pair {
   .month,
   .year {
     @extend %inputs-halves;
 
-    &:last-of-type{
+    &:last-of-type {
       float: right;
     }
   }
 }
 
-.minute{
+.minute {
   float: right;
 }
 
-.datetime{
-  &.month, &.day, &.year{
+.datetime {
+  &.month,
+  &.day,
+  &.year {
     margin-bottom: $base-margin;
   }
 }
 
-/*-------------- Credit Card images ---------- */
-.cc-type{
+/* -------------- Credit Card images ---------- */
+.cc-type {
   width: 38px;
   height: 19px;
   display: inline-block;
 }
 
-.type-visa{
+.type-visa {
   background: url(image-path("cc-type-visa.png")) no-repeat;
   background-size: contain;
   background-position: center center;
 }
 
 .type-mc,
-.type-mastercard{
+.type-mastercard {
   background: url(image-path("cc-type-mc.png")) no-repeat;
   background-size: contain;
   background-position: center center;
 }
 
 /* The vendor prefixed placeholders have to be declared as separate selectors.
 When a browser doesn't understand a selector it invalidates the entire line */
-input::-webkit-input-placeholder{
+input::-webkit-input-placeholder {
   color: darken($smoke, 5%);
 }
 
-input:-moz-placeholder{
+input:-moz-placeholder {
   color: darken($smoke, 5%);
 }
 
-input::-moz-placeholder{
+input::-moz-placeholder {
   color: darken($smoke, 5%);
 }
 
-input[type='text'],
-[type='password'],
-[type='date'],
-[type='datetime'],
-[type='datetime-local'],
-[type='month'],
-[type='week'],
-[type='email'],
-[type='number'],
-[type='search'],
-[type='tel'],
-[type='time'],
-[type='url'],
-[type='color'],
+input[type="text"],
+[type="password"],
+[type="date"],
+[type="datetime"],
+[type="datetime-local"],
+[type="month"],
+[type="week"],
+[type="email"],
+[type="number"],
+[type="search"],
+[type="tel"],
+[type="time"],
+[type="url"],
+[type="color"],
 textarea,
-select{
+select {
   margin-bottom: 0;
   -webkit-appearance: none;
   -moz-appearance: none;
   appearance: none;
 }
 
-select.has-placeholder:nth-child(1){
```

Comment:
Why was this removed?

Reply:
oh, because I realized this was something that I added a long time ago but it doesn't work as expected/we don't use it anywhere